diff --git a/alteracoes.patch b/alteracoes.patch
index 09b5aa5..e69de29 100644
--- a/alteracoes.patch
+++ b/alteracoes.patch
@@ -1,80 +0,0 @@
-diff --git a/src/components/projetos/style.js b/src/components/projetos/style.js
-index af1cf67..db04652 100644
---- a/src/components/projetos/style.js
-+++ b/src/components/projetos/style.js
-@@ -27,18 +27,16 @@ export const GridContainer = styled.div`
-     max-width: 100%;
-     background-color: #ffffff;
-     display: flex;
-+    border-top: 1px solid #ddd;
- 
-   & .sub-item {
-     display: flex;
-     flex-direction: column;
--    justify-content: space-evenly;
--    margin: 0 0 0 20px;
-+    justify-content: space-between;
-+    margin: 0 0 0 15px;
- 
--    & .item-h2, .item-p {
--    color: #323F4B;
--  }
-     & .item-p {
--      font-size: 20px;
-+      font-size: 16px;
-     }
-   };
-   };
-@@ -57,9 +55,10 @@ export const GridContainer = styled.div`
-     
- 
-   /* Transforma somente em telas maiores */
--  @media (min-width: 700px) { /* Ajuste o valor conforme necessário */
-+  @media (min-width: 750px) { /* Ajuste o valor conforme necessário */
-     &:hover {
-       transform: translateY(-30px);
-+      
-     }
-   }
- };
-diff --git a/src/pages/projetos/index.jsx b/src/pages/projetos/index.jsx
-index 963ab67..9bd3d0d 100644
---- a/src/pages/projetos/index.jsx
-+++ b/src/pages/projetos/index.jsx
-@@ -36,7 +36,7 @@ import HeaderLayout from "@/components/layout/headerLayout";
- import MainContentLayout from "@/components/layout/contentLayout";
- import Overlay from "@/components/overlay";
- 
--import { GridContainer } from "@/components/projetos/style";
-+import { GridContainer, P } from "@/components/projetos/style";
- 
- // Definindo o estilo de cada tag de tecnologia
- const tagStyle = {
-@@ -108,15 +108,15 @@ const Projetos = ({ projetos }) => {
-                         />
-                         <div className="item">
-                           <div className="sub-item">
-+                            <p>{projeto.estrelas}</p>
-                             <div className="tech">
--                              <h1 className="item-h1" style={{ display: "inline" }}>Tech:</h1>
-+                              <p ><strong>Tech:</strong></p>
-                               {projeto.tech.map((tech, idx) => (
-                                 <span key={idx} style={tagStyle}>{tech}</span>
-                               ))}
-                             </div>
--                            <h2 className="item-h2">{projeto.title}</h2>
-+                            <p><strong>{projeto.title}</strong></p>
-                             <p className="item-p">{projeto.descricao}</p>
--                            <p>{projeto.estrelas}</p>
-                           </div>
-                         </div>
-                       </Link>
-@@ -160,7 +160,7 @@ export async function getStaticProps() {
-           ],
-           descricao: "Plataforma de cursos online.",
-           estrelas: "⭐⭐⭐⭐⭐ 🚀",
--          alt: "curso online"
-+          alt: "curso online",
-         },
-         {
-           title: "Data Analysis Tendency",
diff --git a/package.json b/package.json
index a0724f8..0546e1f 100644
--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
     "next-sitemap": "^4.2.3",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-icons": "^5.5.0",
     "sharp": "^0.33.5",
     "styled-components": "^6.1.13"
   },
diff --git a/src/components/buildProject/buildProjectMainContainer.js b/src/components/buildProject/buildProjectMainContainer.js
deleted file mode 100644
index 1a81e98..0000000
--- a/src/components/buildProject/buildProjectMainContainer.js
+++ /dev/null
@@ -1,35 +0,0 @@
-import styled, { css } from "styled-components";
-
-export const BuildProjectMainContainer = styled.div`
-
-/* user-select: none; */
-padding: 20px 10px;
-
-& .h3Center-buildProject {
-  margin-bottom: 15px;
-  padding-bottom: 15px;
-}
-
-& .h3Center-buildProject:hover {
-  color: blue; /* azul mais escuro ao passar mouse */
-}
-
-& .h3Center-buildProject.active {
-  color: blue;
-  text-decoration: none; /* pode tirar o underline se quiser */
-}
-`;
-
-export const CollapsibleContent = styled.div`
-  overflow: hidden;
-  max-height: 0;
-  opacity: 0;
-  transition: max-height 0.5s ease, opacity 0.5s ease;
-
-  ${({ isOpen }) =>
-    isOpen &&
-    css`
-      max-height: 9999px; /* valor muito alto para conter todo conteúdo */
-      opacity: 1;
-    `}
-`;
\ No newline at end of file
diff --git a/src/components/buildProject/saaSforDoctors/index.jsx b/src/components/buildProject/saaSforDoctors/index.jsx
deleted file mode 100644
index f88f16a..0000000
--- a/src/components/buildProject/saaSforDoctors/index.jsx
+++ /dev/null
@@ -1,146 +0,0 @@
-import {
-  ContainerBuildProject,
-  SectionTitleBuildProject,
-  ParagraphBuildProject,
-  ListBuildProject,
-  ListItemBuildProject,
-  StepNumberBuildProject,
-} from "@/styles/globalStyles";
-
-const SaaSforDoctors = () => {
-  return (
-    <ContainerBuildProject>
-      <SectionTitleBuildProject>Descrição do Projeto</SectionTitleBuildProject>
-      <ParagraphBuildProject>
-        Projeto voltado para médicos que acompanham pacientes em fase de emagrecimento. O SaaS utiliza N8n como motor de automações, permitindo que o médico apenas informe seu número de WhatsApp para começar. As interações com os pacientes são conduzidas por um agente de IA, que realiza agendamentos, acompanhamento e nutre o lead até se tornar paciente.
-      </ParagraphBuildProject>
-
-      <SectionTitleBuildProject>1. Início</SectionTitleBuildProject>
-      <ParagraphBuildProject>
-        SaaS para médicos, onde o N8n está por trás, e o médico vai adicionar somente o número de WhatsApp dele. O projeto é voltado para médicos que acompanham pacientes em fase de emagrecimento. As funções incluem informar os serviços oferecidos, adicionar valores, protocolos, etc. Uma IA vai gerar automaticamente o prompt com os serviços, valores e forma de atendimento focado em conversão para transformar o lead em paciente.
-      </ParagraphBuildProject>
-
-      <SectionTitleBuildProject>2. Agendamento e Interação com a IA</SectionTitleBuildProject>
-      <ParagraphBuildProject>
-        O médico vai adicionar a credencial do Calendário (Cal.com ou Google) e agendar conversando com o agente.
-      </ParagraphBuildProject>
-      <ParagraphBuildProject>
-        O agente de IA vai identificar se o lead agendou ou não a consulta e vai falar com ele várias vezes até que ele se torne paciente. Após se tornar paciente, ele vai receber dicas, notícias, avaliar o atendimento, etc. O agendamento é feito automaticamente pela conversa, onde o agente vai verificar a agenda e informar dois horários para o paciente escolher. Caso não sejam os horários desejados, ele sempre falará de dois em dois horários somente.
-      </ParagraphBuildProject>
-
-      <SectionTitleBuildProject>3. Funcionalidades do Agente</SectionTitleBuildProject>
-      <ListBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>4</StepNumberBuildProject>
-          O agente também precisa entender áudio, imagens e documentos.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>5</StepNumberBuildProject>
-          O agente consegue enviar imagens dentro de um banco de dados do SaaS que são vinculadas aos serviços que ele oferece.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>6</StepNumberBuildProject>
-          Criação automática de grupo de WhatsApp, onde terá o telefone do paciente e outros agentes como psicólogo, nutricionista, etc.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>7</StepNumberBuildProject>
-          Configuração dos agentes para enviar mensagens de forma automática em horários pré-definidos.
-        </ListItemBuildProject>
-      </ListBuildProject>
-
-      <SectionTitleBuildProject>4. CRM - Funcionalidades</SectionTitleBuildProject>
-      <ListBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>8</StepNumberBuildProject>
-          Respostas humanizadas e personalizadas, sem enviar uma mensagem a cada mensagem do lead.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>9</StepNumberBuildProject>
-          Entendimento de áudio, imagem, documento.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>10</StepNumberBuildProject>
-          Envio de imagens (sem ser link).
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>11</StepNumberBuildProject>
-          Integração com calendário e envio de lembretes de consulta.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>12</StepNumberBuildProject>
-          Envio de dicas baseado no cadastro do lead e no que ele gosta.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>13</StepNumberBuildProject>
-          Identificação de se o lead já é paciente ou não.
-        </ListItemBuildProject>
-      </ListBuildProject>
-
-      <SectionTitleBuildProject>5. Agentes Específicos</SectionTitleBuildProject>
-      <ListBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>14</StepNumberBuildProject>
-          Agente psicólogo: envio de mensagens motivacionais diárias.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>15</StepNumberBuildProject>
-          Agente nutricionista: envio de mensagens e recomendações alimentares com base nas fotos do paciente.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>16</StepNumberBuildProject>
-          Agente personal trainer: envio de treinos e mensagens motivacionais para o paciente.
-        </ListItemBuildProject>
-      </ListBuildProject>
-
-      <SectionTitleBuildProject>6. Dashboard e Monitoramento</SectionTitleBuildProject>
-      <ListBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>17</StepNumberBuildProject>
-          Dashboard para monitoramento da quantidade de leads e mensagens enviadas.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>18</StepNumberBuildProject>
-          Check-list semanal para avaliar a semana do paciente.
-        </ListItemBuildProject>
-      </ListBuildProject>
-
-      <SectionTitleBuildProject>7. Funcionalidades Avançadas</SectionTitleBuildProject>
-      <ListBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>19</StepNumberBuildProject>
-          Cadastro de serviços e valores com fotos de antes e depois para o agente enviar automaticamente.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>20</StepNumberBuildProject>
-          Cadastro completo do lead com exames e informações de saúde para o agente usar em sua interação.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>21</StepNumberBuildProject>
-          Integração com Instagram, WhatsApp e outras plataformas para comunicação.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>22</StepNumberBuildProject>
-          Relatório financeiro e gerenciamento de custos com tokens de IA e integração com plataformas de pagamento.
-        </ListItemBuildProject>
-      </ListBuildProject>
-
-      <SectionTitleBuildProject>Resumo das Funcionalidades</SectionTitleBuildProject>
-      <ListBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>23</StepNumberBuildProject>
-          Automatização de tarefas com integração de IA.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>24</StepNumberBuildProject>
-          Integração com sistemas de pagamento, faturamento e emissão de notas fiscais.
-        </ListItemBuildProject>
-        <ListItemBuildProject>
-          <StepNumberBuildProject>25</StepNumberBuildProject>
-          Interface amigável e fácil de operar para o médico e seus pacientes.
-        </ListItemBuildProject>
-      </ListBuildProject>
-    </ContainerBuildProject>
-  );
-};
-
-export default SaaSforDoctors;
diff --git a/src/components/diariodev/diarioDevMainContainer.js b/src/components/diariodev/diarioDevMainContainer.js
deleted file mode 100644
index 57af97c..0000000
--- a/src/components/diariodev/diarioDevMainContainer.js
+++ /dev/null
@@ -1,43 +0,0 @@
-import styled, { css } from "styled-components";
-
-export const DiarioDevMainContainer = styled.div`
-
-width: 100%;
-padding: 20px 10px;
-
-& .h3Center-diarioDev {
-text-align: center;
-margin-bottom: 15px;
-padding-bottom: 15px;
-cursor: pointer;
-user-select: none;
-}
-
-& .h3Center-diarioDev:hover {
-  color: blue; /* azul mais escuro ao passar mouse */
-}
-
-& .h3Center-diarioDev.active {
-  color: blue;
-  text-decoration: none; /* pode tirar o underline se quiser */
-}
-
-& .EJS, .GitHub, .MySQL {
-border-bottom: 1px solid #ccc;
-/* box-shadow: 0 1px 0 0 #eee; */
-}
-`;
-
-export const CollapsibleContent = styled.div`
-  overflow: hidden;
-  max-height: 0;
-  opacity: 0;
-  transition: max-height 0.5s ease, opacity 0.5s ease;
-
-  ${({ isOpen }) =>
-    isOpen &&
-    css`
-        max-height: none; /* permite altura natural do conteúdo */
-      opacity: 1;
-    `}
-`;
\ No newline at end of file
diff --git a/src/components/layout/sideBarLayout/index.jsx b/src/components/layout/sideBarLayout/index.jsx
index 073b4f5..e030495 100644
--- a/src/components/layout/sideBarLayout/index.jsx
+++ b/src/components/layout/sideBarLayout/index.jsx
@@ -109,6 +109,14 @@ const SideBar = () => {
                 Build Project
               </Span>
             </li>
+            <li>
+              <Span
+                onClick={() => handleNavigate('/e-commerce-construction-project')}
+                className={activeLink === '/e-commerce-construction-project' ? 'active' : ''}
+              >
+                E-commerce
+              </Span>
+            </li>
           </ul>
         </Menu>
         <div style={{ width: "260px", border: "solid 1px #243B53", marginTop: "10px", marginBottom: "10px" }}></div>
diff --git a/src/pages/build-project/index.jsx b/src/pages/build-project/index.jsx
index b044565..1cd423e 100644
--- a/src/pages/build-project/index.jsx
+++ b/src/pages/build-project/index.jsx
@@ -10,10 +10,10 @@ import Overlay from "@/components/overlay";
 
 import Title from "@/components/common/title";
 
-import { BuildProjectMainContainer, CollapsibleContent } from "@/components/buildProject/buildProjectMainContainer";
+import { BuildProjectMainContainer, CollapsibleContent, H2 } from "@/components/buildProject";
 
 import SaaSProjectGuide from "@/components/buildProject/saaSProjectGuide";
-import SaaSforDoctors from "@/components/buildProject/saaSforDoctors";
+import SaaSforDoctors from "@/components/buildProject/projectSaaS";
 import TeamHeader from "@/components/buildProject/team";
 import ModularCleanArchitecture from "@/components/buildProject/ModularCleanArchitecture";
 
@@ -36,7 +36,7 @@ const BuildProject = () => {
 
   const sections = [
     { title: "O que é um projeto SaaS?", component: <SaaSProjectGuide /> },
-    { title: "Desenvolvimento de SaaS para Médicos com Plano de Emagrecimento", component: <SaaSforDoctors /> },
+    { title: "Projeto SaaS para Médicos com Plano de Emagrecimento", component: <SaaSforDoctors /> },
     { title: "Team", component: <TeamHeader /> },
     { title: "Guia Completo do Ciclo de Vida do Projeto SaaS Médico: Do Planejamento à Entrega", component: <ModularCleanArchitecture /> },
   ];
@@ -75,6 +75,7 @@ const BuildProject = () => {
             <HeaderLayout></HeaderLayout>
             <MainContentLayout>
               <BuildProjectMainContainer>
+                <H2>Este projeto está em desenvolvimento. Início 22/04/2025</H2>
                 {sections.map((section, index) => (
                   <>
                     <div key={index}>
diff --git a/src/pages/diario-dev/index.jsx b/src/pages/diario-dev/index.jsx
index 1e7825d..be66e7c 100644
--- a/src/pages/diario-dev/index.jsx
+++ b/src/pages/diario-dev/index.jsx
@@ -10,7 +10,7 @@ import Overlay from "@/components/overlay";
 
 import Title from "@/components/common/title";
 
-import { DiarioDevMainContainer, CollapsibleContent } from "@/components/diariodev/diarioDevMainContainer";
+import { DiarioDevMainContainer, CollapsibleContent } from "@/components/diariodev";
 
 import BibliotecasFerramentas from "@/components/diariodev/bibliotecasFerramentas";
 import ArquiteturaMonolitica from "@/components/diariodev/arquiteturaMonolitica";
diff --git a/yarn.lock b/yarn.lock
index 36d1b3e..6a30b35 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -2421,6 +2421,11 @@ react-dom@^18.3.1:
     loose-envify "^1.1.0"
     scheduler "^0.23.2"
 
+react-icons@^5.5.0:
+  version "5.5.0"
+  resolved "https://registry.yarnpkg.com/react-icons/-/react-icons-5.5.0.tgz#8aa25d3543ff84231685d3331164c00299cdfaf2"
+  integrity sha512-MEFcXdkP3dLo8uumGI5xN3lDFNsRtrjbOEKDLD7yv76v4wpnEq2Lt2qeHaQOr34I/wPN3s3+N08WkQ+CW37Xiw==
+
 react-is@^16.13.1:
   version "16.13.1"
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-16.13.1.tgz#789729a4dc36de2999dc156dd6c1d9c18cea56a4"
