diff --git a/src/context/GlobalStateContext.jsx b/src/context/GlobalStateContext.jsx
index 5f5373f..ce7df43 100644
--- a/src/context/GlobalStateContext.jsx
+++ b/src/context/GlobalStateContext.jsx
@@ -5,7 +5,7 @@ const GlobalStateContext = createContext();
 
 // Criando o provider
 export function GlobalStateProvider({ children }) {
-  const [isHamburguerOpen, setIsHamburguerOpen] = useState(false);
+  const [isHamburguerOpen, setIsHamburguerOpen] = useState(true);
   const [isLoaded, setIsLoaded] = useState(false); // Estado global de carregamento
 
   // O primeiro useEffect lida apenas com o redimensionamento.
