diff --git a/src/components/layout/sideBarLayout/index.jsx b/src/components/layout/sideBarLayout/index.jsx
index f1c4bed..8a21aee 100644
--- a/src/components/layout/sideBarLayout/index.jsx
+++ b/src/components/layout/sideBarLayout/index.jsx
@@ -6,7 +6,7 @@ import Image from 'next/image';
 import { FiArrowRight, FiArrowLeft } from "react-icons/fi";
 
 import perfil from '../../../assets/images/perfil.png';
-import { ContentWrapper, ContentNav, Menu, Arrow, DiagonalOne, DiagonalTwo, ImgPerfil, Span, TitleName, SubTitle, TitleMenuSecret, AppContainer, MenuToggleButton, MainContent, SidebarContainer, RotatingWrapper, Face, FrontFace, BackFace, ProfileSection, ProfileImage, Title, Subtitle, Divider, NavMenu, MenuItemSpan, FlipButtonContainer, FlipButton, ScrollableMenuArea } from './styles';
+import { ContentWrapper, ContentNav, Menu, Arrow, DiagonalOne, DiagonalTwo, ImgPerfil, Span, TitleName, SubTitle, TitleMenuSecret, AppContainer, MenuToggleButton, MainContent, SidebarContainer, RotatingWrapper, Face, FrontFace, BackFace, ProfileSection, ProfileImage, Title, Subtitle, Divider, NavMenu, MenuItemSpan, FlipButtonContainer, FlipButton, ScrollableMenuArea, DividerWithText } from './styles';
 
 // --- Ícones (SVGs embutidos) ---
 const MenuIcon = () => (
@@ -92,7 +92,7 @@ const SideBar = () => {
           <SubTitle>Web Developer Frontend</SubTitle>
 
           <ScrollableMenuArea>
-            <div style={{ width: "260px", border: "solid 1px #374151", marginTop: "20px", marginBottom: "10px" }}></div>
+            <DividerWithText>Menu Principal</DividerWithText>
             <Menu>
               <ul>
                 <li>
@@ -119,128 +119,98 @@ const SideBar = () => {
                     Projetos
                   </Span>
                 </li>
+                <li>
+                  <Span
+                    onClick={() => handleNavigate('/build-project')}
+                    className={activeLink === '/build-project' ? 'active' : ''}
+                  >
+                    Build Project
+                  </Span>
+                </li>
+                <li>
+                  <Span
+                    onClick={() => handleNavigate('/e-commerce-construction-project')}
+                    className={activeLink === '/e-commerce-construction-project' ? 'active' : ''}
+                  >
+                    E-commerce
+                  </Span>
+                </li>
+              </ul>
+            </Menu>
+            <DividerWithText>Artigos Técnicos</DividerWithText>
+            <Menu className='text-center-menu-secreto'>
+              <ul>
+                <li>
+                  <Span
+                    onClick={() => handleNavigate('/menu-secreto/resumo-tech')}
+                    className={activeLink === '/menu-secreto/resumo-tech' ? 'active' : ''}
+                  >
+                    Notas Rápidas
+                  </Span>
+                </li>
                 <li>
                   <Span
                     onClick={() => handleNavigate('/diario-dev')}
                     className={activeLink === '/diario-dev' ? 'active' : ''}
                   >
-                    Diário dev
+                    Blog
                   </Span>
                 </li>
                 <li>
                   <Span
-                    onClick={() => handleNavigate('/build-project')}
-                    className={activeLink === '/build-project' ? 'active' : ''}
+                    onClick={() => handleNavigate('/menu-secreto/nodejs-java-spring')}
+                    className={activeLink === '/menu-secreto/nodejs-java-spring' ? 'active' : ''}
                   >
-                    Build Project
+                    NodeJS / Java
+                  </Span>
+                </li>
+                <li>
+                  <Span
+                    onClick={() => handleNavigate('/menu-secreto/node-js-frameworks')}
+                    className={activeLink === '/menu-secreto/node-js-frameworks' ? 'active' : ''}
+                  >
+                    NodeJS Frameworks
                   </Span>
                 </li>
+              </ul>
+              <DividerWithText>Marketing dígital</DividerWithText>
+              <ul>
                 <li>
                   <Span
-                    onClick={() => handleNavigate('/build-app')}
-                    className={activeLink === '/build-app' ? 'active' : ''}
+                    onClick={() => handleNavigate('/menu-secreto/seo-nextjs-guia')}
+                    className={activeLink === '/menu-secreto/seo-nextjs-guia' ? 'active' : ''}
                   >
-                    Build App
+                    SEO NextJS guia
                   </Span>
                 </li>
                 <li>
                   <Span
-                    onClick={() => handleNavigate('/e-commerce-construction-project')}
-                    className={activeLink === '/e-commerce-construction-project' ? 'active' : ''}
+                    onClick={() => handleNavigate('/menu-secreto/analytics-ga4-install')}
+                    className={activeLink === '/menu-secreto/analytics-ga4-install' ? 'active' : ''}
                   >
-                    E-commerce
+                    Integrar Google Analytics 4 (GA4) no Next.js
+                  </Span>
+                </li>
+                <li>
+                  <Span
+                    onClick={() => handleNavigate('/menu-secreto/utm-parameters')}
+                    className={activeLink === '/menu-secreto/utm-parameters' ? 'active' : ''}
+                  >
+                    UTM Parameters
+                  </Span>
+                </li>
+                <li>
+                  <Span
+                    onClick={() => handleNavigate('/menu-secreto/google-Ads')}
+                    className={activeLink === '/menu-secreto/google-Ads' ? 'active' : ''}
+                  >
+                    Google Ads
                   </Span>
                 </li>
               </ul>
             </Menu>
-            <div style={{ width: "260px", border: "solid 1px #374151", marginTop: "20px", marginBottom: "10px" }}></div>
-
-            {mostrarSpans && (
-              <>
-                <TitleMenuSecret>Menu secreto</TitleMenuSecret>
-                <Menu className='text-center-menu-secreto'>
-                  <ul>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/resumo-tech')}
-                        className={activeLink === '/menu-secreto/resumo-tech' ? 'active' : ''}
-                      >
-                        Resumo tech
-                      </Span>
-                    </li>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/nodejs-java-spring')}
-                        className={activeLink === '/menu-secreto/nodejs-java-spring' ? 'active' : ''}
-                      >
-                        NodeJS / Java
-                      </Span>
-                    </li>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/node-js-frameworks')}
-                        className={activeLink === '/menu-secreto/node-js-frameworks' ? 'active' : ''}
-                      >
-                        NodeJS Frameworks
-                      </Span>
-                    </li>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/configurar-projeto-app')}
-                        className={activeLink === '/menu-secreto/configurar-projeto-app' ? 'active' : ''}
-                      >
-                        Config Projeto APP
-                      </Span>
-                    </li>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/configurar-projeto-api')}
-                        className={activeLink === '/menu-secreto/configurar-projeto-api' ? 'active' : ''}
-                      >
-                        Config Projeto API
-                      </Span>
-                    </li>
-                  </ul>
-                  <TitleMenuSecret>Marketing dígital</TitleMenuSecret>
-                  <ul>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/seo-nextjs-guia')}
-                        className={activeLink === '/menu-secreto/seo-nextjs-guia' ? 'active' : ''}
-                      >
-                        SEO NextJS guia
-                      </Span>
-                    </li>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/analytics-ga4-install')}
-                        className={activeLink === '/menu-secreto/analytics-ga4-install' ? 'active' : ''}
-                      >
-                        Integrar Google Analytics 4 (GA4) no Next.js
-                      </Span>
-                    </li>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/utm-parameters')}
-                        className={activeLink === '/menu-secreto/utm-parameters' ? 'active' : ''}
-                      >
-                        UTM Parameters
-                      </Span>
-                    </li>
-                    <li>
-                      <Span
-                        onClick={() => handleNavigate('/menu-secreto/google-Ads')}
-                        className={activeLink === '/menu-secreto/google-Ads' ? 'active' : ''}
-                      >
-                        Google Ads
-                      </Span>
-                    </li>
-                  </ul>
-                </Menu>
-              </>
-            )}
           </ScrollableMenuArea>
-          <div style={{ width: "260px", border: "solid 1px #374151", marginBottom: "10px" }}></div>
+          {/* <div style={{ width: "100%", border: "solid 1px #374151", marginBottom: "10px" }}></div> */}
           <FlipButtonContainer>
             <FlipButton primary onClick={() => setIsFlipped(true)}><FlipIcon /><span>Ver Extras</span></FlipButton>
           </FlipButtonContainer>
@@ -250,70 +220,39 @@ const SideBar = () => {
             <FiArrowLeft size={30} color='#f1f1f1'></FiArrowLeft>
           </Arrow>
 
-          <TitleName onClick={cliqueSecreto}>Build App</TitleName>
-          <SubTitle>Do Zero ao Deploy</SubTitle>
-          <div style={{ width: "260px", border: "solid 1px #374151", marginTop: "20px", marginBottom: "10px" }}></div>
+          <TitleName onClick={cliqueSecreto}>Build </TitleName>
+          <SubTitle>App Developer</SubTitle>
           <Menu>
+            <DividerWithText>Aplicação</DividerWithText>
             <ul>
               <li>
                 <Span
-                  onClick={() => handleNavigate('/')}
-                  className={activeLink === '/' ? 'active' : ''}
-                >
-                  Home
-                </Span>
-              </li>
-              <li>
-                <Span
-                  onClick={() => handleNavigate('/sobre')}
-                  className={activeLink === '/sobre' ? 'active' : ''}
-                >
-                  Sobre
-                </Span>
-              </li>
-              <li>
-                <Span
-                  onClick={() => handleNavigate('/projetos')}
-                  className={activeLink === '/projetos' ? 'active' : ''}
-                >
-                  Projetos
-                </Span>
-              </li>
-              <li>
-                <Span
-                  onClick={() => handleNavigate('/diario-dev')}
-                  className={activeLink === '/diario-dev' ? 'active' : ''}
-                >
-                  Diário dev
-                </Span>
-              </li>
-              <li>
-                <Span
-                  onClick={() => handleNavigate('/build-project')}
-                  className={activeLink === '/build-project' ? 'active' : ''}
+                  onClick={() => handleNavigate('/build-app')}
+                  className={activeLink === '/build-app' ? 'active' : ''}
                 >
-                  Build Project
+                  Build App
                 </Span>
               </li>
               <li>
                 <Span
-                  onClick={() => handleNavigate('/build-app')}
-                  className={activeLink === '/build-app' ? 'active' : ''}
+                  onClick={() => handleNavigate('/menu-secreto/configurar-projeto-api')}
+                  className={activeLink === '/menu-secreto/configurar-projeto-api' ? 'active' : ''}
                 >
-                  Build App
+                  Os Pilares de um Backend Robusto
                 </Span>
               </li>
               <li>
                 <Span
-                  onClick={() => handleNavigate('/e-commerce-construction-project')}
-                  className={activeLink === '/e-commerce-construction-project' ? 'active' : ''}
+                  onClick={() => handleNavigate('/menu-secreto/configurar-projeto-app')}
+                  className={activeLink === '/menu-secreto/configurar-projeto-app' ? 'active' : ''}
                 >
-                  E-commerce
+                  Boilerplate do App
                 </Span>
               </li>
             </ul>
+            <DividerWithText>API</DividerWithText>
           </Menu>
-          <div style={{ width: "260px", border: "solid 1px #374151", marginTop: "20px", marginBottom: "10px" }}></div>
+          {/* <div style={{ width: "100%", border: "solid 1px #374151", marginTop: "20px", marginBottom: "10px" }}></div> */}
           <FlipButtonContainer>
             <FlipButton onClick={() => setIsFlipped(false)}><FlipIcon style={{ transform: 'rotateY(180deg)' }} /><span>Voltar</span></FlipButton>
           </FlipButtonContainer>
diff --git a/src/components/layout/sideBarLayout/styles.js b/src/components/layout/sideBarLayout/styles.js
index 8f58a3b..03e91ce 100644
--- a/src/components/layout/sideBarLayout/styles.js
+++ b/src/components/layout/sideBarLayout/styles.js
@@ -239,7 +239,7 @@ export const RotatingWrapper = styled.div`
 export const Face = styled.div`
           position: absolute;
           width: 100%;
-   height: 100%;
+          height: 100%;
           display: flex;
           flex-direction: column;
           align-items: center;
@@ -393,4 +393,31 @@ export const ScrollableMenuArea = styled.div`
   &::-webkit-scrollbar-thumb:hover {
     background-color: #6b7280; /* Cor um pouco mais clara no hover */
   }
+`;
+
+export const DividerWithText = styled.div`
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #94a3b8; /* Cor do texto */
+  font-size: 14px;
+  font-weight: 500;
+  width: 90%; /* Controla a largura total */
+  margin: 25px 0; /* Espaçamento vertical */
+
+  /* A linha antes do texto */
+  &::before,
+  &::after {
+    content: '';
+    flex: 1;
+    border-bottom: 1px solid #374151; /* Cor da linha */
+  }
+
+  /* Espaçamento entre a linha e o texto */
+  &::before {
+    margin-right: 0.5em;
+  }
+  &::after {
+    margin-left: 0.5em;
+  }
 `;
\ No newline at end of file
